devhelp/autogen.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
```

Comment:
The files:
- autogen.sh
- config.status
- configure
- configure.ac
- configure.scan
- libtool
- the whole m4 folder
are not necessary at all, just delete them.

Reply:
ok. done!